client-dashboard/src/components/LandingPage.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react'
+import landingPageTemplate from '../templates/landingPageTemplate'
+import axios from 'axios'
+
+const LandingPage: React.FC = () => {
+  const [link, setLink] = useState("")
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const { data } = await axios.get('api/installation/slackInstallationLink');
```

Comment:
Let's change it to `/api/settings` and add `slackInstallationLink` as a field there